src/main/java/model/producer/ExcavatorProducer.java

```diff
@@ -0,0 +1,20 @@
+package model.producer;
+
+import core.mate.academy.model.Excavator;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer {
+    @Override
+    public List<Excavator> get() {
```

Comment:
The `get` method should not start with an empty line according to the checklist. Please remove the empty line before the method declaration.